src/managers/views/iframe.js

```diff
@@ -146,20 +146,23 @@ class IframeView {
 				// apply the layout function to the contents
 				this.settings.layout.format(this.contents);
 
-				// Expand the iframe to the full size of the content
-				this.expand();
-
-				// Listen for events that require an expansion of the iframe
-				this.addListeners();
+        setTimeout((function() {
```

Comment:
Thank you for your suggestion.

We've tried using Promise + requestAnimationFrame but for some reason, this version also cuts off the pages when navigating.